projects/igniteui-angular/src/lib/grids/resizing/pivot-grid/pivot-resizing.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { ColumnType } from '../../common/grid.interface';
+import { PivotRowHeaderGroupType } from '../../pivot-grid/pivot-grid.interface';
+import { IgxColumnResizingService } from '../resizing.service';
+
+
+/**
+ * @hidden
+ * @internal
+ */
+@Injectable()
+export class IgxPivotColumnResizingService extends IgxColumnResizingService {
+    /**
+     * @hidden
+     */
+    public rowHeaderGroup: PivotRowHeaderGroupType;
+
+    /**
+     * @hidden
+     */
+    public getColumnHeaderRenderedWidth() {
+        return this.rowHeaderGroup.header.nativeElement.getBoundingClientRect().width;
+    }
+
+    protected _handlePixelResize(diff: number, column: ColumnType) {
+        const rowDim = this.rowHeaderGroup.parent.dimension;
+        if (!rowDim) return;
+
+        const currentColWidth = parseFloat(column.width);
+        const colMinWidth = column.minWidthPx;
+        const colMaxWidth = column.maxWidthPx;
+        if (currentColWidth + diff < colMinWidth) {
+            rowDim.width = colMinWidth + 'px';
+        } else if (colMaxWidth && (currentColWidth + diff > colMaxWidth)) {
+            rowDim.width = colMaxWidth + 'px';
+        } else {
+            rowDim.width = (currentColWidth + diff) + 'px';
+        }
+        
+        // Notify the grid to reflow, to update if horizontal scrollbar needs to be rendered/removed.
+        this.rowHeaderGroup.grid.pipeTrigger++;
+        this.rowHeaderGroup.grid.notifyChanges(true);
+    }
+
+    protected _handlePercentageResize(diff: number, column: ColumnType) { }
```

Comment:
The size is resolved to px in the DOM, but it does get recalculated when the grid is resized, for example if dimension width is set to:
 width: '25%',

Then when you resize the window the related dimension also changes size:

![width-%](https://user-images.githubusercontent.com/10397980/150161780-b08c19bc-d437-4e98-b891-10ee7fe735d0.gif)

Not sure why the right border is cut off. I'll log that as a separate issue.

If the width is set to % and then it is resized it will become fixed - it will no longer resize when the container size changes.